stored_requests/backends/http_fetcher/fetcher.go

```diff
@@ -81,6 +81,49 @@ func (fetcher *HttpFetcher) FetchRequests(ctx context.Context, requestIDs []stri
 	return
 }
 
+// FetchAccount retrieves an account configuration
+//
+// Request format is similar to the one for requests:
+// GET {endpoint}?account=["<accountID>"]
+//
+// The endpoint is expected to respond with a JSON map with accountID -> accountObject
+// {
+//   "<accountID>": { ... account data ... }
+// }
+func (fetcher *HttpFetcher) FetchAccount(ctx context.Context, accountID string) (json.RawMessage, error) {
```

Comment:
Please consider adding unit test coverage for this

Reply:
http_fetcher is not production ready without the ability to cache, so I will remove it instead.